test/data/parserWithWorkflowGraphPR.json

```diff
@@ -95,6 +95,18 @@
                 ],
                 "requires": ["~pr"]
             }
+        ],
+        "not_pr_job": [
```

Comment:
Do we use `not_pr_job` in any test cases?

Reply:
`not_pr_job` is not chained job from PR. So, I tested not to create this job when prChain is true.
https://github.com/screwdriver-cd/models/pull/315/files#diff-a1295284aaaac017ed0fc3ccc2fd10c8R945

Reply:
You mean testing with `assert.calledThrice` and  three `calledWith`. make sense.